src/js/dropdown-menu/dropdown-menu-view.js

```diff
@@ -227,7 +229,7 @@ define([
                     event.preventDefault();
                 }
 
-                if (key === constants.keyCodes.tab && $el.hasClass('last')) {
+                if (key === constants.keyCodes.tab && !event.shiftKey && $el[0] === this.$lastItem[0]) {
```

Comment:
Nit: Is it at all possible that these might be null/undefined?  Maybe http://underscorejs.org/#first would be safer?